### DISCUSSION TITLE
Does .predict() also use the best weights?

### DISCUSSION THREAD

**Discussion by kaare-mikkelsen** [Lightning-AI/pytorch-lightning#10795] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
On https://pytorch-lightning.readthedocs.io/en/latest/starter/converting.html, it says that ".test() loads the best checkpoint automatically". Is that also the case for .predict()?

**Comment by rohitgr7** [comment, marked as answer]
yes, by default it does load the best checkpoint if you don't provide the model, you can set it too if you want!
```py
trainer.predict(ckpt_path='best')
```
